src/components/Frame/Frame.scss

```diff
@@ -173,8 +173,11 @@ $skip-vertical-offset: rem(10px);
 
   @include frame-when-nav-displayed {
     .hasTopBar & {
-      top: top-bar-height();
-      z-index: z-index(loading-bar, $fixed-element-stacking-order);
+      top: var(--p-override-zero, top-bar-height());
```

Comment:
Why are we overriding `rem(56px)` with 0?

Reply:
In the new design language, the `Loading` bar moves to the very top of the application. In current-form Polaris, it sits below the `TopBar`. Overriding to `0` moves it into that top position.